test/e2e/data/infrastructure-docker/main/clusterclass-quick-start-runtimesdk.yaml

```
@@ -32,6 +32,19 @@ spec:
             apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
             kind: DockerMachineTemplate
             name: quick-start-default-worker-machinetemplate
+    machinePools:
+    - class: default-worker
+      template:
```

```suggestion
      namingStrategy:
        template: "{{ .cluster.name }}-mp-{{ .machinePool.topologyName }}-{{ .random }}"
      template:
```

This got just merged for machinedeployments on main, let's also add it to machinePool (this test only).